rubrik_cdm/api.py

```diff
@@ -70,7 +79,7 @@ def _common_api(self, call_type, api_version, api_endpoint, config=None,
                 if params is not None:
                     request_url = request_url + "?" + '&'.join("{}={}".format(key, val)
                                                                for (key, val) in params.items())
-                request_url = quote(request_url, '://?=&')
+                request_url = quote(request_url, '://?=&[]')
```

Comment:
Excluding [] from HTML escape codes since this breaks the required syntax for IPv6 URLs